src/Component/CodeEditor/CodeEditor.tsx

```diff
@@ -151,13 +148,11 @@ export class CodeEditor extends React.Component<CodeEditorProps, CodeEditorState
   }
 
   styleFromValue = (value: string) => {
-    const {
-      activeParser
-    } = this.state;
+    const activeParser: any = this.state.activeParser;
     return new Promise((resolve, reject) => {
       if (activeParser) {
-        const StyleParser = activeParser;
-        const parserInstance = new StyleParser();
+        const StyleParserClass = activeParser;
+        const parserInstance = activeParser instanceof Function ? new StyleParserClass() : activeParser;
```

Comment:
This seems to be kind of a loose check. But not blocking actually.

Reply:
See above.